docs/api.md

```diff
@@ -0,0 +1,95 @@
+-   Grundsätzlich: REST-APIs!
+
+-   ID == PrimaryKey oder UUID. Etwas das stabil bleibt
+
+-   "ticketNumber" ist die Bon Nummer die man sonst ausgedruckt bekommen
+    würde. Also etwas für den Menschen einfach Lesbares
+
+APIs für Admin-Interface
+========================
+
+Auth:
+-----
+
+-   Bearer-Token als HTTP-Header?! =\> Erstmal ohne Authentifizierung
+    bauen, aber im Hinterkopf behalten
+
+Authentifizierung durchführen (kommt erst später)
+-------------------------------------------------
+
+HTTP-GET /places/\<place-id\>/bearerToken?password=\<pass\>
```

Comment:
sollte das nicht ein POST sein? Password in URL ist doch keine gute Idee, oder?

Reply:
Aus Security-Sicht ist PW in der URL relativ unbedenklich, ausser halt, wenn jemand die URL kopiert und weiterschickt. Aber viele APIs schicken ihren API Key (der ja eigentlich auch nur ein passwort ist) in der URL mit. Alles ausser dem Hostnamen ist in der Anfrage auch verschlüsselt

Reply:
würde das Passwort nicht in der Browser-History auftauchen?

Reply:
Nein.